pkg/build/gobuild.go

```diff
@@ -90,14 +118,25 @@ func NewGo(options ...Option) (Interface, error) {
 //
 // Only valid importpaths that provide commands (i.e., are "package main") are
 // supported.
-func (*gobuild) IsSupportedReference(s string) bool {
-	p, err := gb.Import(s, gb.Default.GOPATH, gb.ImportComment)
+func (g *gobuild) IsSupportedReference(s string) bool {
+	p, err := g.Import(s)
 	if err != nil {
 		return false
 	}
 	return p.IsCommand()
 }
 
+// Import wraps go/build.Import to handle go modules.
+func (g *gobuild) Import(s string) (*gb.Package, error) {
```

Comment:
It doesn't look like `Import` has to be exported, can we unexport it?

Reply:
Imported it because `import` is not a valid identifier :( I'll fix it